pkg/metrics/metrics.jsx

```diff
@@ -452,13 +453,17 @@ class CurrentMetrics extends React.Component {
         if (this.state.cpuCoresUsed.length > 1) {
             const top_cores = this.state.cpuCoresUsed.map((v, i) => [i, v]).sort((a, b) => b[1] - a[1])
                     .slice(0, 16);
-            cores = top_cores.map(c => <Flex key={c[0]} justifyContent={{ default: 'justifyContentSpaceBetween' }}>
-                <FlexItem>{ cockpit.format(_("Core $0"), c[0]) }</FlexItem>
-                <FlexItem>{c[1]}%</FlexItem></Flex>
-            );
+            cores = (<Grid className='cpu-all' component='dl' span={6} hasGutter>
+                {top_cores.sort((a, b) => a[0] - b[0]).map(c =>
+                    <React.Fragment key={c[0]}>
+                        <GridItem component='dt'>{ cockpit.format(_("Core $0"), c[0]) }</GridItem>
+                        <GridItem component='dd'>{c[1]}%</GridItem>
+                    </React.Fragment>)
+                }
+            </Grid>);
```

Comment:
I hope I got this right. I guess the span isn't needed as I'm overriding it in the CSS anyway. I'll push a fix.